content/3.workflow/2.BNO-onboarding.md

```diff
@@ -9,9 +9,27 @@ The BNO onboarding process consists of the following steps:
 ```mermaid
 
 sequenceDiagram
+     autonumber
 
-    PSP ->> BNO: 1
-    BNO ->> PSP: 2
-    BNO ->> GevamuPaymentGateway: 3
+    %% code block %% 
+    participant psp as PSP
+    box Corda Network
+    participant bno as BNO
+    participant gpg as Gevamu Payment Gateway
+    end
+
+    rect rgb(192,243,218)
+    Note right of psp: Off-chain
+    psp ->> bno: Provides HTTPS endpoints<br/> and assigns Network ID to BNO
+    bno ->> psp: Provides HTTPS endpoints
+    end
+
+    rect rgb(252,212,213)
+    Note right of bno: On-chain
+    bno ->> gpg: BNO sets up the Payments Gateway<br/> as a node within their Corda network<br/> using exchanged configuration details
+    end
```

Comment:
![image](https://github.com/gevamu/gevamu.github.io/assets/51794590/83e31a8e-9c8b-4045-b5bc-2de0d18fd5b6)

It is quite dangerous to use static background color, because mermaid can use different color themes. In this case you should be sure, that both black and white texts will be readable on this background.